lib/fastlane/actions/create_keychain.rb

```diff
@@ -28,6 +28,15 @@ def self.run(params)
         command << " ~/Library/Keychains/#{escaped_name}"
 
         commands << Fastlane::Actions.sh(command, log: false)
+
+        if params[:add_to_search_list]
+          keychains = []
+          keychains += Fastlane::Actions.sh("security list-keychains -d user", log: false).shellsplit unless Helper.test?
```

Comment:
`+=`? Also, why not do that in tests? Tests don't actually execute the `sh` action, you can leave it in :+1: